instaRaider.py

```diff
@@ -239,13 +350,19 @@ def main():
     parser.add_argument('-n', '--num-to-download',
                         help='Number of posts to download', type=int)
     parser.add_argument('-l', '--log-level', help="Log level", default='info')
+    parser.add_argument('-m', '--add_metadata',
```

Comment:
Make sure that the documentation #35 reflects the correct useage Metadata function.  In the PR #34 it uses --add-metadata instead of the correct --add_metadata